### DISCUSSION TITLE
How to create the document name on firebase query?

### DISCUSSION THREAD

**Discussion by MikeYates** [Budibase/budibase#11767] [Help] [answered]
I am trying to create a query that names the document id with a specific value of my binding, I tried by manually inputting it in the field with `id` or `docname` none does seem to work, document name is always randomly generated by firebase. When randomly generated, the document does contain `id` field containing the randomly generated name.

**Comment by melohagan** [comment, marked as answer]
Hey @MikeYates 

There is a related issue open. Currently you cannot create a firestore doc with a custom ID.

https://github.com/Budibase/budibase/issues/7047#issuecomment-1699414882
